### PR TITLE
Sprint 9 Fixes

### PR DESCRIPTION
**Pull request by burgwyn**
This addresses:
#763
#764 
#765
#766 
#767
#768
#769 
#772

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -55,11 +55,9 @@ export class Header extends Component {
         <div className="usa-navbar">
           <button className="usa-menu-btn">Menu</button>
           <div className="usa-logo" id="logo">
-            <div className="usa-logo-text">
```
Comment (mjoyce91) [inline src/Components/Header/Header.jsx]: Removing this class breaks some styles in responsive views.

With class:
screen shot 2017-10-19 at 12 20 31 pm

Without class:
screen shot 2017-10-19 at 12 20 25 pm

File: src/login/index.jsx [hunk 2]
```diff
@@ -22,8 +22,8 @@ export class Login extends Component {
 
   // Remember, Redux Form passes the form values to our handler
   // In this case it will be an object with `username` and `password`
-  submit(values) {
-    this.props.loginRequest(values);
+  submit({ username, password }) {
```
Comment (mjoyce91) [inline src/login/index.jsx]: 👍

File: src/login/index.test.jsx [hunk 3]
```diff
@@ -29,18 +29,6 @@ describe('Login', () => {
     expect(wrapper).toBeDefined();
   });
 
-  it('can call the submit function', () => {
```
Comment (mjoyce91) [inline src/login/index.test.jsx]: Why'd we remove this test?

File: src/sass/_details.scss [hunk 4]
```diff
@@ -128,5 +128,5 @@
 .offset-bid-button-container {
   left: 63%;
   position: absolute;
-  top: 79%;
+  top: 85%;
```
Comment (mjoyce91) [inline src/sass/_details.scss]: This works, but there's probably a way to get the button consistently on the middle of the edge regardless of the parent container's height (see my comment on  #768).

File: src/login/index.test.jsx (hunk 3 above)
Comment (burgwyn) [reply src/login/index.test.jsx]: The test right below is similar and more substantive

Comment (mjoyce91) [reply src/login/index.test.jsx]: Ah didn't see that. Yeah that makes sense to test the actual interactions.

**Pull request merged**